src/masks/InteractionMasks.tsx

```diff
@@ -114,7 +114,7 @@ export default function InteractionMasks<R, SR>({
   function getEditorPosition() {
     if (!canvasRef.current) return null;
     const { left, top } = canvasRef.current.getBoundingClientRect();
-    const { scrollTop: docTop, scrollLeft: docLeft } = document.scrollingElement || document.documentElement;
+    const { scrollTop: docTop, scrollLeft: docLeft } = document.scrollingElement!;
```

Comment:
Do we need `!`? Can it be null?

Reply:
![image](https://user-images.githubusercontent.com/567105/78945979-7d6c2800-7ab9-11ea-895b-1d588693cfd9.png)

Unlikely at best. 🤔

Reply:
Although we are not supporting IE, I think it is okay to add back `document.scrollingElement || document.documentElement;` so rdg is somewhat usable in IE. It is not a big change anyway